Blish HUD/Controls/HotkeyAssigner.cs

```diff
@@ -112,7 +112,7 @@ public class HotkeyAssigner : Control {
                             );
 
             // Easy way to get a string representation of the hotkeys
-            string hotkeyRep = string.Join(" + ", _hotkeyDefinition.Keys);
+            string hotkeyRep = string.Join(" + ", _keyBindingDefinition.PrimaryKey);
```

Comment:
shouldnt this be `string hotkeyRep =_keyBindingDefinition.GetBindingDisplayText()`?

Reply:
Yes, you're right - good catch!

I think I'm going to go ahead and update the PR to not include these controls at all since they aren't working correctly anyways and are waiting on the changes introduced in the mvp-ui branch.  I can reintroduce them at that time.